ui/src/app/components/help-sidebar.tsx

```diff
@@ -97,10 +98,16 @@ const styles = reactStyles({
     backgroundColor: 'transparent',
     outline: 'none',
   },
-  highlighted: {
-    color: colorWithWhiteness(colors.success, -0.4),
-    backgroundColor: colorWithWhiteness(colors.success, 0.7),
-    display: 'inline-block'
+  footer: {
+    position: 'absolute',
```

Comment:
Thinking out Loud: I wonder if we could do this with flex as opposed to absolute/relative positions. Probably not important, just a thought exercise.

Reply:
For the pages with enough help content that it scrolls within the sidebar, we want the support footer to stay fixed at the bottom and the content to scroll behind it. Not sure flex would do that, although it's possible as I haven't used it very extensively. 
![Screen Shot 2019-09-26 at 8 07 49 AM](https://user-images.githubusercontent.com/40036095/65690548-e8ca3d00-e034-11e9-8a68-24fa40c0d3f3.png)